### PR TITLE
[DPR-429] Fix title alignment and page size picker

### PR DESCRIPTION
**Pull request by stalwartsolutions**

### CODE REVIEW

**Review by carwynmoj — APPROVED** [review]

File: server/views/partials/layout.njk [hunk 1]
```diff
@@ -32,5 +32,6 @@
   <body>, to avoid blocking the initial render. #}
   <script src="/assets/govuk/all.js"></script>
   <script src="/assets/govukFrontendInit.js"></script>
+  <script src="/assets/app.js"></script>
```
Comment (gavriil-g-moj) [inline server/views/partials/layout.njk:35]: Looking at the above file shouldn't the path be /assets/js/app.js ?

Comment (stalwartsolutions) [reply server/views/partials/layout.njk:35]: That's a fair point. And yet it works for this and the other ones. There must be an alias or similar in place.

Comment (stalwartsolutions) [reply server/views/partials/layout.njk:35]: Here it is: https://github.com/ministryofjustice/hmpps-digital-prison-reporting-mi-ui/blob/main/server/middleware/setUpStaticResources.ts#L41

Comment (gavriil-g-moj) [reply server/views/partials/layout.njk:35]: 🤷‍♂️

**Pull request merged**